examples/nodejs/aws/eventbridge/README.md

```diff
@@ -10,43 +10,43 @@ The app can be used to create, update and delete items in a DynamoDB table and t
 - HTTP API endpoint the same region as Momento API Key. You can copy the endpoint from the console after creating the API Key or refer to the [Regions Section here in the documentation](https://docs.momentohq.com/topics/develop/api-reference/http-api#regions)
```

Comment:
you can delete this file now (assuming your README.template.md has basically the same content) - it will be generated in CI.